src/components/ConversationsSelect.ts

```diff
@@ -16,13 +23,21 @@ export const ConversationsSelect: FC<
   children,
   initialConversation,
   actionId,
+  filter,
   responseUrlEnabled = false,
 }) => {
   const select: ConversationsSelectSpec = {
     type: 'conversations_select',
     initial_conversation: initialConversation,
     action_id: actionId,
     response_url_enabled: responseUrlEnabled,
+    ...(filter && {
+      filter: {
+        include: filter.include,
+        exclude_bot_users: filter.excludeBotUsers,
```

Comment:
having these nested filter fields be undefined is not an issue for slack API?

Reply:
it didn't cause any issues in my testing. they appear to treat it as false